infra-templates/k8s/34/README.md

```diff
@@ -3,15 +3,12 @@
 ### Software Versions
 
 * Kubernetes v1.7.4
-* Etcd v2.3.7
-
-### Upgrading to this Version
-
-Warning: The existing template version _must be_ `v1.2.4-rancher9` or later. Ignoring this will result in data loss. For older templates, please first upgrade to `v1.5.4-rancher1`.
+* Etcd v3.0.17
 
 ### Changelog for Kubernetes 1.7.4
 
-* Added healthcheck to addon-starter
+* Upgraded etcd to v3 API including bugfixes, replicated backups, and automated restore process
```

Comment:
Can you add a line that talks about which service changes and what image it is now? Similar to this: https://github.com/rancher/rancher-catalog/tree/v1.6-development/infra-templates/network-services/24#dns-rancherdnsv0153